cppcache/include/geode/PdxWriter.hpp

```diff
@@ -400,6 +400,11 @@ class APACHE_GEODE_EXPORT PdxWriter {
       std::shared_ptr<PdxUnreadFields> unread) = 0;
 
   virtual std::shared_ptr<PdxSerializer> getPdxSerializer() const = 0;
+
+  virtual int32_t getBytesWritten() { return m_bytesWritten; }
```

Comment:
Why are we introducing this new method into the public API? If it is a size value it should be of `size_t`.

Reply:
Hmm, this was supposed to have been removed from PdxWriter, let me have a look and see what happened.  I think that "removes" it from the public API, but maybe only sort of....

As for why it's there: you and I had a conversation about this one when I first did this work, although that was admittedly a good long while ago.  The new method cleans up this particular bit of mental gymnastics:

```
    if (cacheImpl != nullptr) {	
      uint8_t* stPos = const_cast<uint8_t*>(output.getBuffer()) +	
                       ptc.getStartPositionOffset();	
      int pdxLen = PdxHelper::readInt32(stPos);	
      cachePerfStats.incPdxSerialization(	
          pdxLen + 1 + 2 * 4);  // pdxLen + 93 DSID + len + typeID	
    }
```

Reply:
Hmm... yeah I don't recall. It should probably have some documentation associated with it so we know what the purpose is.